WcaOnRails/app/models/user.rb

```diff
@@ -65,7 +65,7 @@ def wca_id_is_unique_or_for_dummy_account
       user = User.find_by_wca_id(wca_id)
       # If there is a non dummy user with this WCA ID, fail validation.
       if user && !user.dummy_account?
-        errors.add(:wca_id, I18n.t('users.errors.unique'))
+        errors.add(:wca_id, I18n.t('users.errors.unique', email: user.email))
```

Comment:
Yeah, this is something I wondered as well. Maybe @viroulep can provide advice on what's best here? Basically, when we change a string, how will translators know that we did so? Or should we consider strings to be immutable, and always rename the keys when we do want to change a string?